src/filePush/filePush.ts

```diff
@@ -0,0 +1,25 @@
+import Client from 'ssh2-sftp-client';
+
+export const filePush = async (filename: string) => {
+
+  const config = {
+    host: process.env.SFTP_Address,
+    username: process.env.SFTP_User,
+    password: process.env.SFTP_Password,
+  };
+  
+  const sftp = new Client();
+
+  const remoteFolderLocation = '/path/to/save';
+
+  const remoteFileLocation = remoteFolderLocation + filename;
+
+  await sftp.connect(config).then(() => {
+    return sftp.fastPut(filename, remoteFileLocation);
+  }).then(() => {
+    return sftp.end();
+  }).catch((err) => {
```

Comment:
It's potentially worth adding a retry on this `sftp.connect` method in case of any timeouts. There's a package called p-retry which handles this. It's used in a couple of our lambdas but this is one example https://github.com/dvsa/cvs-tsk-update-test-stations/blob/develop/src/crm/getTestStation.ts

Reply:
It'd also be worth logging the error with `logger.error('', error)`